tests/componentTests/podcasts/test.httpRequests.spec.js

```diff
@@ -0,0 +1,91 @@
+const supertest = require('supertest')
+const express = require('express')
+const router = require('../../../routes')
+const { badFieldsMockPodcast, mockPodcast, badMockUpdate, mockNewPodcast } = require('./mockPodcast.js')
+const bodyParser = require('body-parser')
+const app = express()
+
+app.use(router)
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
+
+jest.mock('../../../models/fileModel', () => ({
+  getItem: (id) => id === 1132 ? mockPodcast : null,
+  deleteItem: (id) => Promise.resolve(),
+  updateItem: (id, updatedPodcast) => Promise.resolve(),
+  saveItem: (podcast) => Promise.resolve(),
+  getMaxItem: () => Promise.resolve()
+}))
+
+describe('HTTP requests component test', () => {
+  describe('`POST` method Succeeds to post new podcast', () => {
+    it('calls post /podcast/new, should save mockNewPodcast to database', async () => {
+      await supertest(app)
+        .post('/podcast/new')
+        .send(JSON.parse(JSON.stringify(mockNewPodcast)))
+        .expect(200, 'Podcast has been successfully added')
+    })
+  })
+  describe('`POST` method Fails to post podcast request', () => {
```

Comment:
You don't need to put every single test under it's own describe block. only a bunch of tests that are related

Reply:
What's the reason of the failure here? it should be a part of the title, for example : "it should return 500 when podcast object has fields that don't exist. even though I'm confused- should this return 400 in case of invalid fields?